BUILDING.md

```diff
@@ -174,7 +160,6 @@ cmake --build build
 cmake --install build
 cmake --build build --target openassetio-python-venv
 source build/dist/bin/activate
-pip install ./openassetio
 pip install -r openassetio/tests/python/requirements.txt
 pytest openassetio/tests/python
```

Comment:
[plan] We should def make sure in #624 we switch this to `ctest` as we now have more than just `pytest` tests (we didn't when we wrote this before).

Reply:
We removed the whole section in d95c55ba